libraries/AC_AttitudeControl/AC_PosControl.cpp

```diff
@@ -81,7 +81,6 @@ const AP_Param::GroupInfo AC_PosControl::var_info[] = {
     // @Range: 0.5 5
```

Comment:
I think we can remove this parameter meta data for the PSC_ACC_XY_FILT parameter as well.

Reply:
Done